src/core/opamSystem.ml

```diff
@@ -1637,7 +1637,9 @@ let gpatch = lazy begin
   let rec search_gpatch = function
     | [] -> None
     | patch_cmd::patch_cmds ->
-      match OpamProcess.run (make_command ~name:"patch" patch_cmd ["--version"]) with
+      match OpamProcess.run
+              (make_command ~verbose:OpamCoreConfig.(!r.verbose_level >= 3)
+                 ~name:"patch" patch_cmd ["--version"]) with
```

Comment:
It doesn't feel like a good to stop having `-vv` display _all_ commands run... is there anywhere else we already do this?

Reply:
I don't think this change should be part of this PR

Reply:
I've removed that part from the PR. Feel free to revert to the previous commit if you prefer to have the previous version

Reply:
After checking, not really, the other instances are for verbose level >= 2. The only instance where it is >=3 is for commands run by depext, to avoid the big log.